packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/EditView/index.js

```diff
@@ -266,18 +273,33 @@ const ApiTokenCreateView = () => {
                   }
                   primaryAction={
                     canEditInputs && (
-                      <Button
-                        disabled={isSubmitting}
-                        loading={isSubmitting}
-                        startIcon={<Check />}
-                        type="submit"
-                        size="S"
-                      >
-                        {formatMessage({
-                          id: 'global.save',
-                          defaultMessage: 'Save',
-                        })}
-                      </Button>
+                      <Flex justifyContent="center">
+                        <ButtonWithRightMargin
+                          disabled={isSubmitting}
+                          loading={isSubmitting}
+                          startIcon={<Refresh />}
+                          type="button"
+                          size="S"
+                          variant="tertiary"
+                        >
+                          {formatMessage({
+                            id: 'global.regenerate',
```

Comment:
I'm not sure `Regenerate` is generic enough to live in the `global` NS. What do you think about scoping this to whatever you are using for the other API token related strings for now?

Reply:
ok

Reply:
done